src/Resources/translations/SonataClassificationBundle.en.xliff

```diff
@@ -222,6 +222,18 @@
                 <source>Options</source>
                 <target>Options</target>
             </trans-unit>
+            <trans-unit id="form.label_translation_domain">
+              <source>form.label_translation_domain</source>
+              <target>Translation domain</target>
```

Comment:
Could you please add french translations?
"Translation domain" > "Domaine de traduction"
"Icon" > "Icône"
"CSS Class" > "Classe CSS"